app/styles/styles.css

```diff
@@ -7875,7 +7875,7 @@ span.required {
 
 #mifos-reskin-ui-container {
   position: absolute;
-  top: 0;
+  top: 52px;
```

Comment:
@abhaychawla style changes should be done in the SASS files under `styles-dev` directory.

after doing the changes, just compile it using the command once you cd into styles-dev

`sass styles.scss > ../styles/styles.css` this would generate the style.css file

Reply:
@gkrishnan724 Okay, I will make the required changes

Reply:
@gkrishnan724  Made the changes but on the compilation of sass files to css files, a few lines extra lines were modified. Please have a look and let me know if any other change is required.